ModerationPolicy.md

```diff
@@ -0,0 +1,356 @@
+# Moderation policy
+
+If you are not a member and wish to submit a
+moderation request, please see [Requesting Moderation][].
+
+## Table of Contents
+
+-   [Applicability][]
+-   [Terms][]
+-   [Grounds for Moderation][]
+-   [Requesting Moderation][]
+-   [Consideration of Intent][]
+-   [Guidelines and Requirements][]
+    -   [Collaborator Posts][]
+    -   [Non-Collaborator Posts][]
+    -   [Temporary Interaction Limits][]
+    -   [Temporary and Indefinite Blocks][]
+-   [Privacy of the expressjs/moderation Repository][]
+-   [Moderation Team][]
+-   [Escalation of Issues][]
+-   [Modifications to This Policy][]
+
+## Applicability
+
+This policy applies to all repositories under these GitHub organizations: expressjs, jshttp, and pillarjs, as well as all Working Groups.
+
+Any posts, comments, issues, or pull requests that violate the [Code of Conduct][] are subject to Moderation as defined in this policy.
+
+## Terms
+
+-   _Collaborator_ refers to any individual with configured triage role or higher
+    in any expressjs GitHub organization repository. See
+    [GitHub's Repository roles documentation][] for more information.
+-   _TC_ refers to the [expressjs Technical Committee][].
+-   _Post_ refers to the content and titles of any issue, pull request, comment,
+    discussion, or wiki page.
+-   _Moderate_ means to modify, lock, or delete one or more Posts to correct or
+    address Code of Conduct violations.
+-   _Remove_ refers to the act of removing the configured write (commit)
+    permissions for an individual Collaborator's GitHub account from _all_
+    expressjs, jshttp, and pillarjs repositories as well as removing the account from
+    the expressjs, jshttp, and pillarjs organizations membership.
+-   _Block_ refers to the act of prohibiting an individual GitHub account from any
+    further participation in the expressjs, jshttp, and pillarjs organizations. A block may be
+    _temporary_ or _indefinite_.
+    -   This Moderation Policy applies only to blocking from the organization.
+        Individuals may choose to
+        [block other individuals from their personal GitHub accounts][]. This policy
+        does not restrict blocking from personal GitHub accounts.
+-   _Requester_ refers to an individual requesting Moderation on a Post.
+
+## Grounds for Moderation
+
+Any Post in violation of the expressjs [Code of Conduct][] is
+subject to Moderation.
+
+The Moderation Team is responsible for deciding what constitutes inappropriate
+behavior that may be subject to Moderation.
```

Comment:
this style (that we use in the cpc repo) of breaking only after a full sentence reduces diff churn, and soft-wraps in everyone's editors so it's often easier to edit. if you go for it, best to do it throughout.

```suggestion
Any Post in violation of the expressjs [Code of Conduct][] is subject to Moderation.

The Moderation Team is responsible for deciding what constitutes inappropriate behavior that may be subject to Moderation.
```

Reply:
Thanks, will do that throughout.